aries-site/src/pages/components/skeleton.mdx

```diff
@@ -15,16 +15,11 @@ import { DashBoardSkeletonExample, SkeletonAnatomy } from '../../examples';
   <DashBoardSkeletonExample />
 </Example>
 
-### When to use
-Skeletons are used on all pages to provide a visual layout of the content being loaded.  They help users perceive a loading period as being shorter in duration.
+### Use When
+- Waiting for content to be returned and displayed to a user, to provide a sense of page structure and minimize layout shifts.
 
-### When not to use
-Skeletons should not be used if:
-- The data or elements being loaded are unknown.
-- The user is awaiting feedback for in-context actions such as submitting a form or fetching search results.
-  - The Spinner component may be more appropriate.
-- The loading process takes more than a few minutes to load.
-  - Pair with a message or notification to provide the user with better visibility of a system’s status.
+#### Don't Use When
```

Comment:
Is it worthwhile stating "Don't use for known, static content. Display them immediately while dynamic content is being retrieved."? 

I'm fine with leaving out to keep things simple. We only add it in in the future if needed (e.g. we see skeleton being applied to titles, headings, navigation, etc.). Thoughts?

```suggestion
#### Don't Use When
- Known, static content such as page titles, section headings, and navigational elements which are always shown and independent of data-driven content.
```